scripts/fontawesome/icons.lua

```diff
@@ -0,0 +1,679 @@
+local icons = {
+    ICON_GLASS = '\u{f000}',
```

Comment:
I dont know if you generated this somehow or where did you get this, but if you rename the enum like I wrote, remove the ICON_ prefix here please (so it doesnt look silly... IconGlyphs.ICON_GLASS is kinda weird)

Reply:
Bonus points if you could make it like 

ICON_GLASS -> Glass
ICON_TH_LARGE -> ThLarge

so it fits more with a style of our other enums